test/journal/create_new_entry_test.js

```diff
@@ -85,6 +85,37 @@ describe("Journal", function () {
             return expect(createPatientEntry({ mood: "Happy!" })).to.be.a.journal.createSuccess;
         });
 
+        //moodEmoji
+        it("doesn't require a mood Emoji", function () {
+            return expect(createPatientEntry({ moodEmoji: undefined })).to.be.a.journal.createSuccess;
+        });
+        it("rejects a blank mood Emoji", function () {
+            return expect(createPatientEntry({ moodEmoji: "" })).to.be.an.api.error(400, "invalid_emoji");
+        });
+        it("allows a null mood Emoji", function () {
+            return expect(createPatientEntry({ moodEmoji: null })).to.be.a.journal.createSuccess;
+        });
+        it("allows a unicode mood Emoji", function () {
+            return createPatientEntry({moodEmoji : "\\U0001F625"})
+            .then(function (response){
+                expect(response).to.be.a.journal.createSuccess;
+                expect(response.body.moodEmoji).to.match(/^\\U[\da-f]{1,8}$/i);
```

Comment:
The flow at the moment here is: creating a new journal entry with mood emoji `"\\U0001F625"`, and then checking that the mood emoji in the API response is any valid unicode char code. What you want to check instead is that the mood emoji in the API response is the one you passed in to the request. So `+                expect(response.body.moodEmoji).to.match("\\U0001F625");`. Otherwise, the API could always set the mood emoji to `"\\U00000000"` and this test would still pass.